pyseventeentrack/profile.py

```diff
@@ -80,13 +80,22 @@ async def packages(
             if last_event_raw:
                 event = json.loads(last_event_raw)
 
+            timestamp = event.get("a")
+            dd = event.get("dd")
+            if dd:
+                try:
+                    dt_str = f"{dd['d']}T{dd['t']}{dd['tz']}"
```

Comment:
```suggestion
                    dt_str = f"{dd['d']}T{dd['t']}{dd.get('tz') or ''}"
```
I noticed in some scenarios that the tz can be null in the json object, so maybe this is better to avoid a datetime parsing exception? (the result should be the same either way as `timestamp` would preserve its initial `event.get("a")` value which it this case would be identical to the one from the formatted string)